src/select-travel-token-screen/SelectTravelTokenScreenComponent.tsx

```diff
@@ -72,9 +71,8 @@ export const SelectTravelTokenScreenComponent = ({onAfterSave}: Props) => {
   );
 
   // Filter for unique travel rights config types
-  const availableFareContractsTypes = availableTravelRights
-    .filter(onlyUniquesBasedOnField('type'))
```

Comment:
This is the only place we use the `travelRight.type` field, and seems to be a (small) performance optimization. Removing it should have no functional change. We're also talking about removing the `requiresTokenOnMobile` option, which should remove the need for all of this code.